src/dialect.js

```diff
@@ -244,7 +249,7 @@ class Dialect {
    * @return Function        Returns the quoter handler.
    */
   quoter(quoter) {
-    if (arguments.length) {
+    if (quoter) {
```

Comment:
this is a behavior change I think. Passing an empty string as the quoter wouldn't work...not sure we'd ever want to do that or not though?

Reply:
is needs to be a function. so maybe event check if its a function or throw if not

Reply:
yea, I'd do that. Better developer experience IMO